### PR TITLE
TRUNK-5189: Used daemon user as instructed in TODO section

### PR DESCRIPTION
**Pull request by iranganN**
https://issues.openmrs.org/browse/TRUNK-5189

### CODE REVIEW

**Comment by dkayiwa** [comment]
@iranganN did you look into the codacy review comments?

File: api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java [hunk 1]
```diff
@@ -198,9 +253,22 @@ public void notifySuperUsers(String messageCode, Exception cause, Object... mess
 		//If there is not user creator for the alert ( because it is being created at start-up )create a user
 		//TODO switch this to use the daemon user when ticket TRUNK-120 is complete
 		if (alert.getCreator() == null) {
-			alert.setCreator(new User(1));
+			
+			try{
+				JdbcConnection connection = (JdbcConnection) DatabaseUpdater.getConnection();
```
Comment (wluyima) [inline api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java]: Why don't you go through the AlertService instead of raw JDBC?

**Comment by wluyima** [comment]
Can you remove the unused imports please as codacy is suggesting?

**Comment by dkayiwa** [comment]
@iranganN are you still working on this?

**Comment by iranganN** [comment]
I was busy during the last few days. I will take a look at the errors

File: api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java [hunk 2]
```diff
@@ -162,6 +169,45 @@ public void purgeAlert(Alert alert) throws APIException {
...
+			} else {
+				log.warn("No row returned by getInt() method");
+			}
+			
+			if (rs.next()) {
+				log.warn("Multiple rows returned by getInt() method");
+			}
+			
+			return result;
+		}
```
[the remark this answers is not in the document]
Comment (dkayiwa) [reply api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java]: @iranganN did you see the above codacy comment?

**Comment by dkayiwa** [comment]
@iranganN ping me to reopen this when you resume working on it.

**Pull request closed without merging**